src/amo/pages/UsersUnsubscribe/index.js

```diff
@@ -44,12 +44,16 @@ type Props = {|
   |},
 |};
 
+type PropsFromState = {|
+  isUnsubscribed: boolean | null | void,
```

Comment:
That's not how the previous type was defined. Why does it need to be `null` or `void`, and if that's the case, then maybe we could use a maybe type `?boolean`.

Reply:
According to the [docs in the reducer](https://github.com/mozilla/addons-frontend/blob/master/src/amo/reducers/users.js#L110-L113), all three are possible values and each means something different.